tools/cell_census_builder/validate.py

```diff
@@ -69,12 +70,12 @@ def validate_all_soma_objects_exist(soma_path: str, experiment_builders: List[Ex
     with soma.Collection.open(soma_path, context=SOMA_TileDB_Context()) as census:
         assert census.soma_type == "SOMACollection"
         assert soma.Collection.exists(census.uri)
-        assert "cxg_schema_version" in census.metadata and census.metadata["cxg_schema_version"] == CXG_SCHEMA_VERSION
-        assert (
-            "census_schema_version" in census.metadata
-            and census.metadata["census_schema_version"] == CENSUS_SCHEMA_VERSION
-        )
-        assert "created_on" in census.metadata and datetime.fromisoformat(census.metadata["created_on"])
+        assert "cxg_schema_version" in census.metadata
+        assert census.metadata["cxg_schema_version"] == CXG_SCHEMA_VERSION
+        assert "census_schema_version" in census.metadata
```

Comment:
nano-nit:  as noted earlier, this is redundant (ie. the next line will generate a KeyError if not present)

Reply:
all cleaned up